### PR TITLE
Clickable Summary Items

### PR DESCRIPTION
**Pull request by ShrayRastogi**
1. Updated GameServerBuildsSummaryItem to be clickable and navigate to respective build details
2. Updated ClustersSummaryItem to be clickable and navigate to respective clsuter details

### CODE REVIEW

**Comment by ShrayRastogi** [comment]
@dgkanatsios @javier-op - Please review the PR, thanks!

**Comment by ShrayRastogi** [comment]
@dgkanatsios @javier-op - live demo here - http://4.224.15.5:3000 😊

File: src/Home/Home.tsx [hunk 1]
```diff
@@ -35,15 +44,20 @@ function Home({ clusters }: HomeProps) {
...
+        if (!perBuildMetadata[buildName]) {
+          perBuildMetadata[buildName] = emptyMetadata();
+        }
         total.standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         perCluster[clusterName].standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         perBuild[buildName].standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         total.active += gsb.status.currentActive ? gsb.status.currentActive : 0;
         perCluster[clusterName].active += gsb.status.currentActive ? gsb.status.currentActive : 0;
         perBuild[buildName].active += gsb.status.currentActive ? gsb.status.currentActive : 0;
+        perBuildMetadata[buildName].namespace = gsb.metadata.namespace ? gsb.metadata.namespace : "";
```
Comment (javier-op) [inline src/Home/Home.tsx:56]: Hello and thanks for looking into the project! I was out yesterday, so I couldn't explain some stuff.

So, here when I group the data by buildName it's actually from all clusters, that means that if two different clusters have a build with the same name the data gets aggregated, so linking to a specific cluster wouldn't be correct. The idea of this is that the user might have the same title in different regions and wants to see an overall status.

Looking back, the UI does not make it clear that we are aggregating the data like this, so there is some room for improvement here. A simple solution would be to just split the item into the actual builds from each cluster and change the key to something like `[buildName]:[cluster]`, but of course feel free to try something else.

Comment (ShrayRastogi) [reply src/Home/Home.tsx]: @javier-op - I have made the changes as per our discussion.

**Comment by javier-op** [comment]
Hello, I just tested this and everything looks fine. Unfortunately, it looks like it breaks the test for the home page.

If you go to `src/tests/Home.tests.tsx` you can go to these lines
image
 you will see that I expect a row of the builds table to be like "test_gameserverbuild1 4 4", with the addition of the new column it should be like "test_gameserverbuild1 cluster1 4 4".

**Comment by ShrayRastogi** [comment]
> Hello, I just tested this and everything looks fine. Unfortunately, it looks like it breaks the test for the home page.
> 
> If you go to `src/tests/Home.tests.tsx` you can go to these lines image you will see that I expect a row of the builds table to be like "test_gameserverbuild1 4 4", with the addition of the new column it should be like "test_gameserverbuild1 cluster1 4 4".

@javier-op - sure, will fix it, thanks!

**Comment by javier-op** [comment]
Hey, I found why the tests keeps failing, that "test_gameserverbuild1 4 4" row was actually adding the data from two clusters, it had 1 1 on cluster1 and 3 3 on cluster2, and we just changed that behavior. So, we have to split it in the test too:

```
before:
  const build1String = "test_gameserverbuild1 4 4";
  const build1Row = await screen.findByRole("row", { name: new RegExp(build1String) });
  expect(build1Row).toBeInTheDocument();

after:
  const build11String = "test_gameserverbuild1 cluster1 1 1";
  const build11Row = await screen.findByRole("row", { name: new RegExp(build11String) });
  expect(build11Row).toBeInTheDocument();

  const build12String = "test_gameserverbuild1 cluster2 3 3";
  const build12Row = await screen.findByRole("row", { name: new RegExp(build12String) });
  expect(build12Row).toBeInTheDocument();
```

**Comment by ShrayRastogi** [comment]
> Hey, I found why the tests keeps failing, that "test_gameserverbuild1 4 4" row was actually adding the data from two clusters, it had 1 1 on cluster1 and 3 3 on cluster2, and we just changed that behavior. So, we have to split it in the test too:
> 
> ```
> before:
>   const build1String = "test_gameserverbuild1 4 4";
>   const build1Row = await screen.findByRole("row", { name: new RegExp(build1String) });
>   expect(build1Row).toBeInTheDocument();
> 
> after:
>   const build11String = "test_gameserverbuild1 cluster1 1 1";
>   const build11Row = await screen.findByRole("row", { name: new RegExp(build11String) });
>   expect(build11Row).toBeInTheDocument();
> 
>   const build12String = "test_gameserverbuild1 cluster2 3 3";
>   const build12Row = await screen.findByRole("row", { name: new RegExp(build12String) });
>   expect(build12Row).toBeInTheDocument();
> ```

Hey thanks Javier for pointing it out, yeah I understood that we would now have two rows for same build per cluster that's why did not add the other one, but forgot to split the numbers 😄

**Review by javier-op — APPROVED** [review]
:shipit:

**Pull request merged**